### PR TITLE
Fix/profilepicture

### PR DESCRIPTION
**Pull request by luismolinuevo**
I branched off of Charlie/to do branch. My task was to fixed the images no appearing in the letter/[id]. I fixed some issues. First the profile pictures weren't saving/returning the incorrect link(not the photo link). Then I made sure that the photo_uri is consistent in letterhome, profile, letter/[id]. Then I created to profileimage component that could be used throughout the codebase.

### CODE REVIEW

File: components/general/ProfileImage.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <div>
+      <div className={`${sizeClass} bg-gray-200 rounded-full flex items-center justify-center overflow-hidden mr-2`}>
+        {photo_uri && photo_uri.length != 0 ? (
+          <img
+            src={photo_uri}
+            class="w-full h-full object-cover"
+            alt="profile picture"
+            // width={24}
+            // height={24}
+            // layout="responsive"
```
Comment (madarez) [inline components/general/ProfileImage.jsx:17]: @luismolinuevo , do we need the comments here?

File: app/letterhome/page.js [hunk 2]
```diff
@@ -113,15 +117,16 @@ export default function Home() {
...
+												{/* <div className="w-8 h-8 bg-gray-200 rounded-full flex items-center justify-center overflow-hidden mr-2"> */}
+													{/* {rec?.photo_uri ? (
+														<img src={rec?.photo_uri} class="w-full h-full object-cover" alt="profile picture" />
 													) : (
 														<span className="text-xl text-gray-600">
 															{rec?.first_name?.[0]}
 														</span>
-													)}
-												</div>
+													)} */}
```
Comment (madarez) [inline app/letterhome/page.js:127]: @luismolinuevo , do we need the comments here?

File: app/edit-profile-user-image/page.js [hunk 3]
```diff
@@ -77,35 +79,77 @@ export default function EditProfileUserImage() {
...
+  //           await updateDoc(doc(db, "users", uid), {
+  //             photo_uri: storageUrl,
+  //           });
+
+  //           router.push("/profile");
+  //         }
+  //       }
+  //     );
+  //   }
+  // };
```
Comment (madarez) [inline app/edit-profile-user-image/page.js:107]: @luismolinuevo , do we need the comments here?

File: app/edit-profile-user-image/page.js [hunk 4]
```diff
@@ -132,5 +176,5 @@ export default function EditProfileUserImage() {
         </div>
```
Comment (madarez) [inline app/edit-profile-user-image/page.js:176]: We have two Save buttons when you save the image:
Screenshot 2024-09-08 at 11 21 59 PM
In this situation, maybe we want to update the second button here to something `Done` or `Back` and change its function to be as such. What do you think?

Comment (luismolinuevo) [reply app/edit-profile-user-image/page.js:176]: I would remove one. If not remove the page entirely.

**Review by madarez — APPROVED** [review]

**Pull request merged**